webui/react/package-lock.json

```diff
@@ -22750,7 +24380,7 @@
     },
     "node_modules/omnibar": {
       "version": "2.3.5",
-      "resolved": "https://github.com/hamidzr/omnibar.git#688f7240d23e27c037f7fd49df92cf238474503c",
+      "resolved": "git+ssh://git@github.com/hamidzr/omnibar.git#688f7240d23e27c037f7fd49df92cf238474503c",
```

Comment:
should be fixed

Reply:
ah it seems it was caught by ci as well as expected I was worried it wasn't https://circleci.com/gh/determined-ai/determined/869755?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link